helper/schema/grpc_provider_test.go

```diff
@@ -2319,7 +2319,13 @@ func TestStopContext_stop(t *testing.T) {
 				}
 				close(doneCh)
 			}()
-			server.StopProvider(context.Background(), &tfprotov5.StopProviderRequest{})
+
+			_, err = server.StopProvider(context.Background(), &tfprotov5.StopProviderRequest{})
+
+			if err != nil {
+				t.Fatalf("unexpected StopProvider error: %s", err)
+			}
```

Comment:
For consistency in this changeset:

```suggestion
			if _, err := server.StopProvider(context.Background(), &tfprotov5.StopProviderRequest{}); err != nil {
				t.Fatalf("unexpected StopProvider error: %s", err)
			}
```